### PR TITLE
fix: node:util instead of util

### PR DESCRIPTION
**Pull request by mertcanaltin**
@anonrig after your suggestion, I updated these
https://github.com/nodejs/undici/pull/2938#discussion_r1529551347

### CODE REVIEW

**Review by ronag — APPROVED** [review]

File: test/fetch/request-inspect-custom.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 const { describe, it } = require('node:test')
 const assert = require('assert')
```
Comment (Uzlopak) [inline test/fetch/request-inspect-custom.js:4]:
```suggestion
const assert = require('node:assert')
```

File: test/fetch/response-inspect-custom.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 const { describe, it } = require('node:test')
 const assert = require('assert')
```
Comment (Uzlopak) [inline test/fetch/response-inspect-custom.js:4]:
```suggestion
const assert = require('node:assert')
```

Comment (mertcanaltin) [reply test/fetch/response-inspect-custom.js:4]: thankyou

**Review by Uzlopak — APPROVED** [review]
Lgtm

**Pull request merged**